crawler/rss.py

```diff
@@ -16,26 +16,37 @@ def preprocess(cls, source, options):
         return [RSSConfig.links[source]]
     
     @classmethod
-    def extract(cls, objs, options):
+    def extract(cls, objs, code, options):
         """
         Extract news from BeautifulSoup object.
         """
         # TODO Optimize to each newspaper style
         soup = objs[0]
-        ret = list()
-        for entry in soup.findAll("entry"):
+        ret = list()      
+
+        for entry in soup.findAll(RSSConfig.item_name[code]):
             temp = dict()
```

Comment:
rename "temp" to something more meaningful. Each variable, method, etc need to be self-explanatory, and shouldn't require reading the code to understand what they do. 

Is each "temp" dictionary an article? Then let's call it article.